crucible-llvm/src/Lang/Crucible/LLVM/Intrinsics.hs

```diff
@@ -1450,6 +1451,25 @@ llvmPutsOverride =
   (KnownBV @32)
   (\memOps sym args -> Ctx.uncurryAssignment (callPuts sym memOps) args)
 
+llvmStrlenOverride
+  :: (IsSymInterface sym, HasPtrWidth wptr, wptr ~ ArchWidth arch)
+  => LLVMOverride p sym arch (EmptyCtx ::> LLVMPointerType wptr) (BVType 64)
```

Comment:
This should really use the `sizeT` combinators rather than fixing the size to 64 bits.